### ISSUE TITLE
Pose2Mesh: Graph Convolutional Network for 3D Human Pose and Mesh Recovery from a 2D Human Pose

### ISSUE THREAD

**Issue by Arata2887** [Arata2887/issue_copy#33]
## 0.どんな論文? /What does this work do?

- https://arxiv.org/pdf/2008.09047v3
- ECCV 2020
- proposes Pose2Mesh, a novel GraphCNN-based（graph convolutional neural network） system that estimates 3D human pose and mesh vertices directly from a 2D human pose.

## 1. 論文のモチベーション / Motivation(s) of the paper / document

- Appearance domain gap problem: Differences in image appearance between controlled environments (e.g., laboratory) and in-the-wild environments can hinder model performance.
- Representation issues of 3D rotations: Existing methods(SMPL-based) suffer from non-unique and non-minimal representation problems in pose parameters.
- Pose2Mesh using a model-free approach, that estimates the 3D coordinates of human mesh vertices directly from the 2D human pose to avoid the representation issues, while fully exploiting the mesh topology using a GraphCNN in a coarse-to-fine manner.  
### Pose2Mesh

> <img width="610" alt="Fig 1" src="https://github.com/Arata2887/issue_copy/assets/119403000/a8e5198a-139a-4bd7-b812-8ca52d87fca3">

**PoseNet:** converts the input 2D human pose into a 3D human pose by utilizing the geometric information of the 2D pose, outputting a root-relative 3D pose.

**MeshNet:** converts the input 2D human pose into a 3D human pose by utilizing the geometric information of the 2D pose, outputting a root-relative 3D pose.

> <img width="610" alt="Fig 2" src="https://github.com/Arata2887/issue_copy/assets/119403000/a6f62093-186e-4d48-86aa-c1fd5d77a0c0">

- **Mesh Coarsening:** Starting from the initial mesh GM0 (12288 vertices), it continuously merges child vertices to parent vertices, generating coarser and coarser meshes, until the coarsest mesh GM7(96 vertices) is produced.

- **Adding Fake Nodes:** For each generated coarse graph, fake nodes without edges are added to ensure the stability of the graph structure during the coarsening process.

### Quantitative results

**Table 5** shows the accuracy comparison between state-of-the-art methods and
Pose2Mesh on **Human3.6M**(The dataset names on top are training sets.).

> <img width="473" alt="Table 5" src="https://github.com/Arata2887/issue_copy/assets/119403000/cdf7cc18-d8ef-44a2-962e-4ede77e602bd">

**Table 6** shows the accuracy comparison between state-of-the-art methods and
Pose2Mesh on **3DPW**(The dataset names on top are training sets.).

> <img width="617" alt="Table 6" src="https://github.com/Arata2887/issue_copy/assets/119403000/4a58dea3-8f73-464a-89d1-2b8a3bec4d7f">

### Qualitative results

> <img width="601" alt="Fig 4" src="https://github.com/Arata2887/issue_copy/assets/119403000/2d7b2e26-2a46-451a-bfd0-3fd0bbb80517">

> <img width="602" alt="Fig 5" src="https://github.com/Arata2887/issue_copy/assets/119403000/92c5fb47-24d0-4bd9-9a35-f411baf3d7dc">

> <img width="531" alt="Fig 6" src="https://github.com/Arata2887/issue_copy/assets/119403000/d361d3c8-1769-4ed6-88c7-1b1ad68fcb16">

> <img width="600" alt="Fig 7" src="https://github.com/Arata2887/issue_copy/assets/119403000/3240fbdb-8d61-46d7-b481-75aaa85ed061">

### Ablation studies

> <img width="598" alt="Table 1,2,3" src="https://github.com/Arata2887/issue_copy/assets/119403000/b572699f-e477-4734-a2d0-a74dde4d2266">

## 4.公開コードやデータセットは? /Public codes or datasets?

- https://github.com/hongsukchoi/Pose2Mesh_RELEASE
